samples/test/tables.test.js

```diff
@@ -272,6 +319,20 @@ describe('Tables', () => {
     assert.match(output, /Inserted 2 rows/);
   });
 
+  it(`copy multiple source tables to a given destination`, async () => {
+    execSync(`node createTable.js ${datasetId} destinationTable "${schema}"`);
+    const output = execSync(
+      `node copyTableMultipleSource.js ${datasetId} ${tableId} destinationTable`
+    );
+    assert.include(
```

Comment:
I don't think this does what you want it to 🙃  This statement:

```js
'sourceTable' && 'destinationTable' && 'createDisposition' && 'writeDisposition'
```

Will always evaluate to `writeDisposition`.  JavaScript does weird truthy checks with `&&` that leads to this - try it in the repl!

You probably want to write 4 separate `assert.include` statements

Reply:
Gotcha.